PythonScripts/Movement/Helper/KinematicModelHelper.py

```diff
@@ -1,13 +1,5 @@
-import numpy
-import numpy as np
-from enum import Enum
+from numpy import numpy, ndarray
```

Comment:
cannot import name 'numpy' from 'numpy'

```suggestion
import numpy
from numpy import ndarray
```